.github/workflows/ci.yml

```diff
@@ -1,7 +1,7 @@
 name: CI
 
 env:
-  go-version: '1.19'
+  go-version: "1.19"
```

Comment:
but why is this 1.19?

Reply:
I tried 1.18. Using it breaks the Windows tests with the following errors:

`ThreadSanitizer failed to allocate 0x000000959000 (9801728) bytes at 0x200d8fbb54000 (error code: 87)`

See e.g. here: https://github.com/spacemeshos/post/actions/runs/3236096321/jobs/5301430586 with 1.19 this does not happen. If this is a go 1.18 issue or caused by the GCC that is used by Github I'm not sure about. See also https://github.com/golang/go/issues/46099